### PR TITLE
fix  requre test in openshift

### PR DESCRIPTION
**Pull request by jscotka**

### CODE REVIEW

**Comment by jscotka** [comment]
```
 PYTHONPATH=/src-packit-service PYTHONDONTWRITEBYTECODE=1 python3 -m pytest --color=yes --verbose --showlocals --cov=packit_service --cov-report=term-missing ./tests/unit ./tests/integration/
2019-11-28 14:04:37.216500 | test-node | [1m============================= test session starts ==============================[0m
2019-11-28 14:04:37.216892 | test-node | platform linux -- Python 3.7.5, pytest-3.10.1, py-1.7.0, pluggy-0.8.1 -- /usr/bin/python3
2019-11-28 14:04:37.216915 | test-node | cachedir: .pytest_cache
2019-11-28 14:04:37.216925 | test-node | rootdir: /src-packit-service, inifile: pytest.ini
2019-11-28 14:04:37.216948 | test-node | plugins: cov-2.6.0, celery-4.2.1
2019-11-28 14:04:38.258167 | test-node | [1mcollecting ... [0mcollected 47 items / 1 errors
2019-11-28 14:04:38.539526 | test-node |
2019-11-28 14:04:38.539591 | test-node | ==================================== ERRORS ====================================
2019-11-28 14:04:38.539821 | test-node | ________________ ERROR collecting tests/unit/test_copr_build.py ________________
2019-11-28 14:04:38.539862 | test-node | [31mImportError while importing test module '/src-packit-service/tests/unit/test_copr_build.py'.
2019-11-28 14:04:38.539873 | test-node | Hint: make sure your test modules/packages have valid Python names.
2019-11-28 14:04:38.539879 | test-node | Traceback:
2019-11-28 14:04:38.539885 | test-node | tests/unit/test_copr_build.py:7: in <module>
2019-11-28 14:04:38.539890 | test-node |     from packit.config import (
2019-11-28 14:04:38.539899 | test-node | E   ImportError: cannot import name 'JobNotifyType' from 'packit.config' (/usr/local/lib/python3.7/site-packages/packit/config/__init__.py)[0m
2019-11-28 14:04:38.540341 | test-node |
2019-11-28 14:04:38.540361 | test-node | ----------- coverage: platform linux, python 3.7.5-final-0 -----------
```

**Comment by jscotka** [comment]
``
mypy.....................................................................Failed
hookid: mypy

files/scripts/clean_up_empty_tasks.py:16: error: Argument "port" to "Redis" has incompatible type "Union[str, int]"; expected "int"
files/scripts/clean_up_empty_tasks.py:18: error: Argument 2 to "getenv" has incompatible type "str"; expected "int"

Check if rebase is needed................................................Passed
``

**Comment by lachmanfrantisek** [comment]
> ImportError: cannot import name 'JobNotifyType' from 'packit.config' (/usr/local/lib/python3.7/site-packages/packit/config/__init__.py)

Should be fixed in https://github.com/packit-service/packit-service/pull/260/files

File: Makefile [hunk 1]
```diff
@@ -57,4 +57,12 @@ check-within-openshift:
 	pytest-3 -k test_update
 
 requre-purge-files:
-	requre-patch purge --replaces "requests.sessions%send:Date:str:Fri, 01 Nov 2019 13-36-03 GMT" --replaces 'requests.sessions%send:ETag:str:W/"1e51b8e1c48787a433405211e9e0fe61"' --replaces "requests.sessions%send:elapsed:float:0.2" --replaces "metadata:latency:float:0" --replaces "requests.sessions%_content:expires_at:str:2019-11-01T14:35:53Z" --replaces "requests.sessions%_content:token:str:v1.1bd88d399b8c70e8b88e22cbdaa72abad5e399db" --replaces ":X-GitHub-Request-Id:str:DE82:1411D:42114C4:4F8EF0E:5DBC34B9" --replaces ":X-RateLimit-Reset:str:1573052591" tests_requre/test_data/test_*/*.yaml
+	requre-patch purge --replaces "requests.sessions%send:Date:str:Fri, 01 Nov 2019 13-36-03 GMT" \
```
Comment (lachmanfrantisek) [inline Makefile]: What about using pre-commit hook so we have it in one place. (You can run `pre-commit run --all requre-purge`.)

Comment (lachmanfrantisek) [reply Makefile]: https://github.com/packit-service/requre/blob/master/.pre-commit-hooks.yaml

Comment (jscotka) [reply Makefile]: Sure, but ideally in new PR. are you okay with this?

Comment (jscotka) [reply Makefile]: Ah, now I understand, that we can remove this target, or better to say, use your mentioned command. makes sesnse

Comment (lachmanfrantisek) [reply Makefile]: Please, update the hook if you need some other replacements.

Comment (jscotka) [reply Makefile]: nope, I hope that these are fine, but question is if we should also remove latency data, to not have differences there.

Comment (lachmanfrantisek) [reply Makefile]: I am ok with seeing the changes on latency.

Comment (jscotka) [reply Makefile]: > I am ok with seeing the changes on latency.

so everything could be fine now, lets wait for seconod PR, and then I'll rebase it and we'll see :-)

**Comment by jscotka** [comment]
> LGTM! very nice work with the pre-commit hook and running the redis-server for tests (except for the sleep 5)

Yep, ``sleep 5`` is not elegant, I'll create issue for this, e.g. this redis could be run as separated pod, and this pod could just use it,

**Pull request merged**